tests/fixtures/jobs/lever-senior-platform.html

```
@@ -0,0 +1,68 @@
+<!doctype html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>Senior Platform Engineer · ExampleCo on Lever</title>
+</head>
+<body>
+<div class="content-wrapper posting-page">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="content-wrapper posting-page"> ] on line 8.

(tag-pair)